### PR TITLE
Update schedule in README.md

### PR DESCRIPTION
**Pull request by maxrjones**
Update conveners and schedule. I will modify the schedule again once we know the final timing.

### CODE REVIEW

**Review by leouieda — APPROVED** [review]
Thanks! Will this be running live streamed? I thought they would just upload the recording to their system.

By the way, we should definitely put this on YouTube after the conference. @weiji14 did they try to get you to sign away copyright for the video?

**Comment by weiji14** [comment]
> Thanks! Will this be running live streamed? I thought they would just upload the recording to their system.

No idea if it's going to be live streamed. But I think having an outline of what's in this short course if helpful, and a general sense of how many minutes each section will take.

> By the way, we should definitely put this on YouTube after the conference. @weiji14 did they try to get you to sign away copyright for the video?

Agreed. I haven't uploaded the video yet, still need to record my intro part and do the final stitching (which might take an hour or two for 1080p60fps :star_struck:). But I'll definitely get it done by end of the day.

File: README.md [hunk 1]
```diff
@@ -24,7 +24,11 @@ short course at [EGU General Assembly 2022](https://www.egu22.eu)!
 
 |  Time (UTC) | Event                                                 |
 |:------------|:------------------------------------------------------|
-| 13:10-14.40 | TODO                                                  |
+| 13:10-13:20 | Introduction                                          |
+| 13:20-13.40 | Anatomy of a PyGMT figure                             |
+| 13:40-14.00 | Integration with the scientific Python ecosystem      |
+| 14:00-14.20 | Making some Mars maps with pygmt                      |
+| 14:20-14.40 | LiDAR Point clouds to 3D surfaces                     |
```
Comment (weiji14) [inline README.md]: Here are the final timings.

```suggestion
| 13:10-13:17 | Introduction                                          |
| 13:17-13.39 | Anatomy of a PyGMT figure                             |
| 13:39-13.55 | Integration with the scientific Python ecosystem      |
| 13:55-14.15 | Making some Mars maps with pygmt                      |
| 14:15-14.40 | LiDAR Point clouds to 3D surfaces                     |
```

**Review by weiji14 — APPROVED** [review]

**Comment by weiji14** [comment]
Oh and feel free to merge after making the commit. The website won't be updated because the GitHub Action is only triggered for changes in the `book/` directory, see https://github.com/GenericMappingTools/egu22pygmt/blob/c4bc1b9ff0539c9f65b991c935e2946101057e55/.github/workflows/deploy-book.yml#L15-L16

File: README.md (hunk 1 above)
Comment (maxrjones) [reply README.md]: Thanks!

**Pull request merged**